nemo/collections/common/metrics/classification_accuracy.py

```diff
@@ -13,37 +13,12 @@
 # limitations under the License.
 
 import torch
-from pytorch_lightning.metrics import TensorMetric
+from pytorch_lightning.metrics import Metric
 
-__all__ = ['TopKClassificationAccuracy', 'compute_topk_accuracy']
+__all__ = ['TopKClassificationAccuracy']
 
 
-def compute_topk_accuracy(correct_counts, total_counts):
```

Comment:
This function is used in asr/models/classification_models.py

Reply:
We might need to create functional metric for this case.

Reply:
The PTL Metrics API is independent of PTL. Take a look at this example from their docs: 
```
from pytorch_lightning import metrics

train_accuracy = metrics.Accuracy()
valid_accuracy = metrics.Accuracy(compute_on_step=False)

for epoch in range(epochs):
    for x, y in train_data:
        y_hat = model(x)

        # training step accuracy
        batch_acc = train_accuracy(y_hat, y)

    for x, y in valid_data:
        y_hat = model(x)
        valid_accuracy(y_hat, y)

# total accuracy over all training batches
total_train_accuracy = train_accuracy.compute()

# total accuracy over all validation batches
total_valid_accuracy = valid_accuracy.compute()
```

Reply:
Yes I understand that, but why do we want to make a basic utility function into something that wraps PTL of any form ? Please revert this one and simply call it from a PTL metric class if needed.